### DISCUSSION TITLE
What is the timezone?

### DISCUSSION THREAD

**Discussion by bilogic** [composer/composer#12252] [Q&A / Support] [answered]
https://getcomposer.org/doc/04-schema.md#time

Format does not accept or specify a timezone, can I assume it is GMT+0? And if so, perhaps add it to the docs?

**Comment by Seldaek** [comment, marked as answer]
Yes, I'd definitely say UTC. Feel free to PR the docs and/or schema file.

**Reply by bilogic** [reply]
filed https://github.com/composer/composer/discussions/12252 thanks
